source/js/components/petition/donation-modal.jsx

```diff
@@ -85,6 +86,8 @@ class DonationModal extends Component {
     if (!this.donateURL) {
```

Comment:
There seems to be no reference in the code base regarding donateURL as a prop. So at the moment this link is always defaulting to `form=donate`. However, I am leaving this here as I can use this prop while implementing #10846

Reply:
Thanks for the note. Either in this PR or in #10846, could we add a method for generating `donateURL`? Right now the variable name `donateURL` could be a bit misleading as it's default to a query string with `?` and later appended with more query params. I think using [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams) would probably be a good/cleaner choice.